feature-libs/quote/quote-aware/component/quote-aware.component.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: './quote-aware.component.html',
+})
+export class QuoteAwareComponent {}
```

Comment:
`query:` Can you remind me what was the idea behind this component and related logic?